src/database/seeders/20200211104631-dummy-user.js

```diff
@@ -85,6 +97,33 @@ export default {
       imageUrl: 'https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.freepik.com%2Ffree-photos-vectors%2Fman-profile&psig=AOvVaw1-_OdOwQ-SYcfMGvGLOYb4&ust=1582456400155000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCOjsvNCD5ecCFQAAAAAdAAAAABAD',
       passportNumber: 'GH1234567',
       bio: 'i love eating a lot. food is bae',
+      notifyInApp: true,
+      notifyByEmail: true,
+      createdAt: new Date(),
+      updatedAt: new Date()
+    },
+    {
+      userID: '0e22ed1c-a1a5-4f49-a4ca-000732bfa49o',
+      firstName: 'Thee',
+      lastName: 'Manager',
+      method: 'local',
+      isVerified: true,
+      isUpdated: true,
+      email: 'itsafact57@gmail.com',
+      password: '$2a$10$JUCGXOZMZUDUHXqRpbdoVuQ.0RuEEV26NKwnZUQJ2K1tE4FwW.nE.',
+      role: 'manager',
+      gender: 'male',
+      preferedLang: 'fr',
+      birthDate: '1998-02-20',
+      preferedCurrency: 'USD',
+      residence: 'kimironko',
+      managerEmail: 'rusimbipatrick@outlook.com',
+      department: 'IT',
+      imageUrl: 'https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.freepik.com%2Ffree-photos-vectors%2Fman-profile&psig=AOvVaw1-_OdOwQ-SYcfMGvGLOYb4&ust=1582456400155000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCOjsvNCD5ecCFQAAAAAdAAAAABAD',
+      passportNumber: 'GH1124457',
+      bio: 'Developer Avocado',
+      notifyInApp: true,
+      notifyByEmail: false,
       createdAt: new Date(),
       updatedAt: new Date()
```

Comment:
@RusimbiP Since there is already a seeded manager we are all using is it possible for you to use that same manager? instead of adding more seeds that might are not necessary.  You can check in this file  `20200211104632-dummy-users.js` for the seeded manager